packages/core/src/helpers/calls.ts

```diff
@@ -12,7 +12,7 @@ export function warnOnInvalidCall(call: Call | Falsy) {
   console.warn(`Invalid contract call: address=${contract.address} method=${method} args=${args}`)
 }
 
-export function encodeCallData(call: Call | Falsy): RawCall | Falsy {
+export function encodeCallData(call: Call | Falsy, chainId: number): RawCall | Falsy {
```

Comment:
encodeCallData

Reply:
options object is for public apis, and this function is internal and only cares about chainId